util/printchplenv

```diff
@@ -231,7 +231,7 @@ def filter_overrides(chpl_env):
 def filter_tidy(chpl_env):
     comm = ENV_VALS['CHPL_COMM']
     if chpl_env.name == '  CHPL_COMM_SUBSTRATE':
-        return comm != 'none' and comm != 'ugni'
+        return comm != 'none' and comm != 'ofi' and comm != 'ugni'
```

Comment:
I think this is a little cleaner as `return comm not in ['none', 'ofi', 'ugni']`. Or we could just switch to `comm == 'gasnet'`